redbot/setup.py

```diff
@@ -414,7 +414,10 @@ async def remove_instance_interaction() -> None:
     "--overwrite-existing-instance",
     type=bool,
     is_flag=True,
-    help=("Overwrites an existing instance."),
+    help=(
+        "Overwrites an existing instance.\n",
+        "Note: Using this can erase all the data related to the existing instance."
```

Comment:
Grammar nitpick that you can ignore if you want:

First "the" is unnecessary

Reply:
Sure, can you add a suggestion, so that I can use the handy little "commit suggestion" button?

Reply:
I cannot unfortunately, that option is only available to org members :(